src/main/java/com/opentok/ArchiveProperties.java

```diff
@@ -165,6 +188,11 @@ public ArchiveLayout layout() {
             valueList.add(name);
             params.put("name", valueList);
         }
+        if (null != resolution) {
```

Comment:
Not trying to be picky here, but I think the format should be `resolution != null` to be consistent. What do you think?